scaaml/io/dataset.py

```diff
@@ -623,23 +625,23 @@ def _check_disjoint_keys(self, pbar, key_ap: str, deep_check: bool = True):
         Raises: ValueError if some key from train is present in test.
         """
         seen_keys = set()
-        for i in range(len(self.shards_list['test'])):
+        for i in range(len(self.shards_list["test"])):
```

Comment:
I thought `range(len(...))` was a discouraged pattern nowadays.
If it is, maybe replace with:
```python
for i, _ in enumarate(self.shards_list[Dataset.TEST_SPLIT]):
```

And in all cases, the text constant should be replaced for consistency